Kernel/Arch/x86_64/ISABus/I8042Controller.cpp

```diff
@@ -19,15 +19,6 @@ UNMAP_AFTER_INIT NonnullLockRefPtr<I8042Controller> I8042Controller::initialize(
     return adopt_lock_ref(*new I8042Controller());
 }
 
-LockRefPtr<MouseDevice> I8042Controller::mouse() const
-{
-    return m_mouse_device;
-}
-LockRefPtr<KeyboardDevice> I8042Controller::keyboard() const
-{
-    return m_keyboard_device;
-}
-
```

Comment:
You should remove these methods in the same commit where you stop using them.

Reply:
Done.

Reply:
This is not the right commit - you removed usage of them in 35bfa4a7538686431db19da03ecc1e17291c28f4